AutoGVP/add_INFO_fields.R

```diff
@@ -0,0 +1,59 @@
+################################################################################
+# add_INFO_fields.R
+# written by Ammar Naqvi
+#
+# This script annotates INFO column variables with variants that have been 
+# processed and re-called by AutoGVP
+#
+# usage: Rscript  add_INFO_fields.R  --input_table <tsv file> 
+#                                       --vcf_file <vcf file> 
+#                                       --out_prefix <string>
+################################################################################
+
+suppressPackageStartupMessages({
+  library("tidyverse")
+  library("optparse")
+  library("vroom")
+  library("vcfR")
+} )
+
+#Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+## set up directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+analysis_dir <- file.path(root_dir, "AutoGVP")
+results_dir <- file.path(analysis_dir, "results")
+input_dir   <- file.path(analysis_dir, "input")
+
+# parse parameters     
+option_list <- list(
```

Comment:
few things:
1. I think that the input table being an output table adds an extra step here - I was thinking that this script would run automatically after either 01 or 02 and that a user does not need to run this separately. 
2. We could call this `--autogvp_calls` or something, but still - can we set this up so there is a bash script to run either 01+03 or 02+03 with all of the required options for both of these things to happen in one run command?